<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppletLogSettingsWidget</class>
 <widget class="QWidget" name="AppletLogSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Log Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="DialogItemsGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="m_SavePushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Save Log Settings</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_CancelPushButton">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>2</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="m_SelectAllLevelsButton">
          <property name="text">
           <string>Check All</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_ClearAllLevelsButton">
          <property name="text">
           <string>Uncheck All</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="m_LogLevelSettingLabel">
          <property name="text">
           <string>Log Level Setting</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="m_LogLevelCheckBox_1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Log Fatal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_LogLevelCheckBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Log Assert</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_LogLevelCheckBox_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Log Severe</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_LogLevelCheckBox_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Log Error</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_LogLevelCheckBox_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Log Warn</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_LogLevelCheckBox_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Log Debug</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_LogLevelCheckBox_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Log Info</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_LogLevelCheckBox_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Log Status</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_LogLevelCheckBox_9">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Log Verbose</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="m_SelectAllModulesButton">
          <property name="text">
           <string>Check All</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_ClearAllModulesButton">
          <property name="text">
           <string>Uncheck All</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="m_LogLevelSettingLabel_2">
          <property name="text">
           <string>Log Modules Setting</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="m_LogModuleCheckBox_1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string notr="true">Text not set</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_LogModuleCheckBox_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string notr="true">Log Connect</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_LogModuleCheckBox_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string notr="true">Log Listen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_LogModuleCheckBox_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string notr="true">Log Sockets</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_LogModuleCheckBox_5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string notr="true">Log Packets</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_LogModuleCheckBox_6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string notr="true">Log Net Access</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_LogModuleCheckBox_7">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string notr="true">Log Network State</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_LogModuleCheckBox_8">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string notr="true">Log Network Manager</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_LogModuleCheckBox_9">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string notr="true">Log Is Port Open</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_LogModuleCheckBox_10">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string notr="true">Log Thread</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_LogModuleCheckBox_11">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string notr="true">Log Storage</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_LogModuleCheckBox_12">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string notr="true">Log Assets</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_LogModuleCheckBox_13">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string notr="true">Log Plugins</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="m_LogModuleCheckBox_14">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string notr="true">Log Window Positions</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_LogModuleCheckBox_15">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string notr="true">Log Startup</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_LogModuleCheckBox_16">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string notr="true">Log Hosts</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_LogModuleCheckBox_17">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string notr="true">Log Player</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_LogModuleCheckBox_18">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string notr="true">Log TCP Data</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_LogModuleCheckBox_19">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string notr="true">Log UDP Data</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_LogModuleCheckBox_20">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string notr="true">Log Fatal</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_LogModuleCheckBox_21">
            <property name="text">
             <string notr="true">CheckBox</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_LogModuleCheckBox_22">
            <property name="text">
             <string notr="true">CheckBox</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_LogModuleCheckBox_23">
            <property name="text">
             <string notr="true">CheckBox</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_LogModuleCheckBox_24">
            <property name="text">
             <string notr="true">CheckBox</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_LogModuleCheckBox_25">
            <property name="text">
             <string notr="true">CheckBox</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_LogModuleCheckBox_26">
            <property name="text">
             <string notr="true">CheckBox</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
